internal/pkg/deploy/deploy_test.go

```diff
@@ -542,3 +542,107 @@ func Test_IsJobDeployed(t *testing.T) {
 		})
 	}
 }
+
+func TestStore_ListDeployedSNSTopics(t *testing.T) {
+	testCases := map[string]struct {
+		inputApp   string
+		inputEnv   string
+		setupMocks func(mocks storeMock)
+
+		wantedError  error
+		wantedTopics []Topic
+	}{
+		"return error if fail to get resources by tag": {
```

Comment:
Should we also add a test that gives an example of how topics without the service tag should not included in the result (yet)?

Reply:
Done! added to the successful test case. I can make it explicit though.